gnome-initial-setup/gis-assistant.c

```diff
@@ -232,6 +232,7 @@ update_navigation_buttons (GisAssistant *assistant)
 
       if (gis_page_get_complete (page)) {
         set_suggested_action_sensitive (next_widget, TRUE);
+	gtk_widget_grab_focus (next_widget);
         set_navigation_button (assistant, next_widget);
```

Comment:
Looks like indentation does not match, make sure to use the same type as the source file

Reply:
Fixed! I will pay more attention to that in the future. Thanks!

Reply:
I actually have no idea what is going on here. Using the github IDE, and VS Code I can't get it to line up.

Reply:
Nvm finally got it